hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/TransactionProcessor.java

```diff
@@ -205,10 +208,34 @@ private InvolvedParties computeInvolvedParties(
         if (transaction.isCreate()) {
             final Address to;
             if (transaction.isEthereumTransaction()) {
+                final ReadableAccountStore accountStore =
+                        updater.enhancement().nativeOperations().readableAccountStore();
+                var createBody = requireNonNull(transaction.hapiCreation());
```

Comment:
Imo it would be much cleaner to include this code directly where the synthetic `ContractCreateTransactionBody` is first built [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/SynthTxnUtils.java#L149). (The `senderId` and `accountStore` should both be available there.)